Add methods to restore nuget packages in vs code article and update c… (#788)
telerik/maui-docs: installation/vs-code-integration/new-project-wizard.md

![Telerik UI for .NET MAUI VS Code snippets](./images/MauiTemplateWizard.png)

## Set up the Telerik NuGet Server

> You can use one of the following options to configure the needed credentials for your telerik license (trial or commersial):

Reviewer: ```suggestion
You can use one of the following options to configure the required credentials for your Telerik license (trial or commercial):
```

### Using the .NET CLI

> When adding NuGet sources from the .NET CLI, the credentials are stored inside the `Nuget.Config` file. To add the Telerik NuGet package source with the .NET CLI, use the `dotnet nuget add source` command. This command creates or updates the `NuGet.Config` file for you, so you don't have to edit it manually. For more details on this case, review the [Restoring NuGet Packages in your CL Workflow]({%slug nuget-keys%}) article.

Reviewer: ```suggestion

Run `dotnet nuget add source`.

When you are adding NuGet sources from the .NET CLI, the credentials are stored inside the `Nuget.Config` file. This command creates or updates the `NuGet.Config` file with your Telerik license credentials for you, so you don't have to edit it manually.

See also: [Restoring NuGet Packages in Your CI Workflow]({%slug nuget-keys%}).
```

> ### Editing the NuGet.Config file

Reviewer: ```suggestion
### Editing the NuGet.Config File
```

> Nuget package sources and other settings are stored in a `NuGet.Config` file. You can read more about the file structure in the Microsoft article: [Nuget.Config References](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file). To edit the file and add the Telerik feed, you need to: 

Reviewer: ```suggestion
NuGet package sources and other settings are stored in a `NuGet.Config` file. You can read more about the file structure in the Microsoft article: [NuGet.Config References](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file). To edit the file and add the Telerik feed, you need to: 
```

> 1. Ensure you editing the correct and desired `.config` file. You can also create a new one with the `dotnet new nugetconfig` command. 

Reviewer: ```suggestion
1. Ensure you are editing the correct and desired `.config` file. You can also create a new one with the `dotnet new nugetconfig` command. 
```

1. Add the Telerik package source to the `.config` file. 

> Example of sample `NuGet.Config` file:

Reviewer: ```suggestion
Here is a sample `NuGet.Config` file:
```

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <packageSources>
    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
    <clear />
    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
    <add key="TelerikOnlineFeed" value="https://nuget.telerik.com/v3/index.json" />
    </packageSources>
    <packageSourceCredentials>
    <TelerikOnlineFeed>

>         <add key="Username" value="<TELERIK EMAIL or api-key>" />

Reviewer: Why is `api-key` lowercase here and uppercase below?

Author: we use this structure, example here: https://docs.telerik.com/devtools/maui/installation/using-nuget-keys#using-a-nugetconfig-file-with-your-projects following this: https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file
